packages/web/src/features/languages/LanguageView.tsx

```diff
@@ -1,11 +1,60 @@
 import { useTranslation } from 'react-i18next';
+import { Link } from '../../shared/components/Link';
+import apiClient from '../../shared/apiClient';
+import { Icon } from '../../shared/components/Icon';
+import {
+  List,
+  ListBody,
+  ListCell,
+  ListHeader,
+  ListHeaderCell,
+  ListRow,
+  ListRowAction,
+} from '../../shared/components/List';
+import View from '../../shared/components/View';
+import ViewTitle from '../../shared/components/ViewTitle';
+import { useLoaderData } from 'react-router-dom';
+import { GetLanguagesResponseBody } from '@translation/api-types';
+import { capitalize } from '../../shared/utils';
+
+export function languagesViewLoader() {
+  return apiClient.languages.findAll();
+}
 
 export default function LanguagesView() {
+  const languages = useLoaderData() as GetLanguagesResponseBody;
+
   const { t } = useTranslation();
 
   return (
-    <div className="absolute w-full h-full flex items-center justify-center">
-      {t('languagesView')}
-    </div>
+    <View fitToScreen>
+      <div className="m-auto w-fit">
+        <ViewTitle>{capitalize(t('language', { count: 100 }))}</ViewTitle>
```

Comment:
In the view title, we want the plural for many languages, so we pass a large count.